articles/concepts/dirac-notation.md

```diff
@@ -6,6 +6,7 @@ uid: microsoft.quantum.concepts.dirac
 ms.author: nawiebe@microsoft.com 
 ms.date: 12/11/2017
 ms.topic: article
+no-loc: ['$$', '$', '\cdots', 'bmatrix', '\ddots', '\equiv', '\sum', '\begin', '\end', '\sqrt', '\otimes', '{', '}', '\text', '\phi', '\kappa', '\psi', '\alpha', '\beta', '\gamma', '\delta', '\omega', '\bra', '\ket', '\boldone', '\\\\', '\\', '=', '\frac', '\text', '\mapsto', '\dagger', '\to', '\begin{cases}', '\end{cases}', '\operatorname', '\braket', '\id', '\expect', '\defeq', '\variance', '\dd', '&', '\begin{align}', '\end{align}', '\Lambda', '\lambda', '\Omega', '\mathrm', '\left', '\right', '\qquad', '\times', '\big', '\langle', '\rangle', '\bigg', '\Big', '|', '\mathbb', '\vec', '\in', '\texttt', '\ne', '<', '>', '\leq', '\geq', '~~', '~']
 ---
```

Comment:
I can't see the preview build, but this is a good solution if it works. The problem I was encountering with the inline tag was that it stripped all the backslashes when converting to the html source -
Markdown =  :::no-loc text="$\ket{n} = \ket{b_{n-1}...b_1b_0}$"::: 
Source = $ket{n} = ket{b_{n-1}...b_1b_0}$

Reply:
I had a chance to look at the preview and it looks fine. I think we should add the string collection to the metadata for any other topics that need it and push through a merge and loc run. 

I'll close out my PR.

Reply:
@KittyYeungQ @Bradben Do we know how often the loc runs happen? E.g. are they automatic when things are pushed to master and/or live?

I would suggest actually holding off on adding the string to other pages for two reasons:
1. in the worst case, at least any potential errors would be limited to these specific pages
2. this current list was made from a quick run through of only the TeX strings used in the Concepts section. 
If it works, I could do a more thorough search through the docs for other symbols/strings that may be used in TeX (e.g. some of the Q# Libraries pages get very deep in mathematical weeds, so there are likely some there that don't appear in Concepts). Then that exhaustive list could be simply added to all the existing pages, as opposed to scanning each one for which symbols are used and which aren't. Thoughts?

Reply:
Sounds good.

Reply:
All the content which is Machine Translated is pushed out for localization daily and it gets localized and handed back daily.

Reply:
@KittyYeungQ So should we proceed with merging this in order to test?

Reply:
Sure.